### DISCUSSION TITLE
More than 1 user for streaming

### DISCUSSION THREAD

**Discussion by parryhotter007** [streamlink/streamlink#5719] [Q&A] [answered]
### Description

Hi. Apologies if it has been asked before. I used the following command to stream a playlist to local server. I want to know, does streamlink support more than 1 user to connect at the same time?

streamlink --player-external-http --player-external-http-port 8080 --default-stream best --http-ignore-env --http-no-ssl-verify --url "http://samplewebsite.com/1234.m3u8"

**Comment by bastimeyer** [comment, marked as answer]
No, this is not supported because the `--player-external-http{,-continuous}` output shares the same code as `--player{,-continuous}-http`, which is basically just a while-loop which waits for the next HTTP connection to be made by a player until the stream input URL gets resolved and data gets written to the HTTP client connection. A second player couldn't work like this, because this would mean that Streamlink would have to implement logic for resolving streams for each new HTTP connection simultaneously.

If you want simultaneous outputs for multiple clients, then you need to spawn multiple streamlink processes.

**Comment by WolfganP** [comment]
> If you want simultaneous outputs for multiple clients, then you need to spawn multiple streamlink processes.

@parryhotter007  or you can pipe streamlink output into ffmpeg or vlc and restream / broadcast.
